source/core.py

```diff
@@ -235,7 +237,6 @@ def onEndSession(evt):
 	log.debug("Initializing GUI")
 	import gui
 	gui.initialize()
-
```

Comment:
This is the only unix style line ending in the file. Replaced with a windows style line ending.